improver/wxcode/wxcode_decision_tree.py

```diff
@@ -231,10 +212,7 @@ def wxcode_decision_tree():
             "probability_thresholds": [0.5, 0.5],
             "threshold_condition": ">=",
             "condition_combination": "AND",
-            "diagnostic_fields": [
-                "probability_of_lwe_precipitation_rate_above_threshold",
-                (LOW_CLOUD_PROB_ABOVE),
-            ],
+            "diagnostic_fields": [PRECIP_PROB_ABOVE, (LOW_CLOUD_PROB_ABOVE),],
```

Comment:
I think we can lose the brackets now we are not mixing strings and variables.

Reply:
I'm a bit scared of editing this as I don't understand what the query interpretation is doing.  Could we have a quick chat so I can tidy up properly?

Reply:
The brackets are just really ancient legacy that has been preserved endlessly from when the string name was really long and had to be wrapped onto multiple lines (before black relaxed the line length condition), e.g.

https://github.com/metoppv/improver/blame/44d2df465b4a0181e214670c888249b93e052bc4/lib/improver/wxcode/wxcode_decision_tree.py#L277

No harm will come of removing them.

Reply:
OK thank you.  I've tidied this up now.